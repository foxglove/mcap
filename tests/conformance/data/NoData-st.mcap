version https://git-lfs.github.com/spec/v1
oid sha256:33429f8c728a3395e7221519f445d4950f0c621c7a7153bd9174fa9c76320c70
size 108
