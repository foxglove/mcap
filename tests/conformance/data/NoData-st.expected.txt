Header profile="" library=""
Statistics messageCount=0 channelCount=0 attachmentCount=0 chunkCount=0 channelMessageCounts={}
Footer summaryStart=38 summaryOffsetStart=0
